guides/line-management/README.md

```diff
@@ -0,0 +1,35 @@
+# Line management at Made Tech
+
+Line managers help others to learn and grow, deliver great results, and progress their careers.
+
+People who report to a line manager are called "direct reports".
+
+This guidance isn't complete yet. If you need help, ask:
+- the people team (**#team-people** on slack) 
+- the line manager community (**#supply-learning-line-managers** on slack)
+- Yasemin Ercan, our Head of People
+
+## What do I need to do as a line manager?
+
+### One-to-one meetings
+Hold regular one-to-one meetings, at least once a month.  
+
+### Probation
+Help your direct reports to pass their probation period.  
+Work with the Market Principal to extend or fail probations, if necessary.
+
```

Comment:
There's something about failing probation is really a sign of poor hiring rather than poor line management (or even poor performance if the issue is more about core values alignment).

Reply:
I like this. I think I would sink this detail into the "more guidance on probation".